src/ADRIA.jl

```diff
@@ -66,6 +66,7 @@ include("interventions/fogging.jl")
 
 include("io/ResultSet.jl")
 include("io/result_io.jl")
+include("io/rme_result_io.jl")
```

Comment:
Standardize file name.

Known / established acronyms should always be capitalized (e.g., rme -> RME)